jabgui/src/main/java/org/jabref/gui/actions/StandardActions.java

```diff
@@ -135,9 +135,11 @@ public enum StandardActions implements Action {
     EXPAND_ALL(Localization.lang("Expand all")),
     COLLAPSE_ALL(Localization.lang("Collapse all")),
 
-    NEW_ENTRY(Localization.lang("New entry"), IconTheme.JabRefIcons.ADD_ENTRY, KeyBinding.NEW_ENTRY),
-    NEW_ARTICLE(Localization.lang("New article"), IconTheme.JabRefIcons.ADD_ARTICLE),
-    NEW_ENTRY_FROM_PLAIN_TEXT(Localization.lang("New entry from plain text"), IconTheme.JabRefIcons.NEW_ENTRY_FROM_PLAIN_TEXT),
+    CREATE_ENTRY_IMMEDIATE(Localization.lang("Add entry"), IconTheme.JabRefIcons.CREATE_ENTRY_IMMEDIATE),
+    CREATE_ENTRY(Localization.lang("Create new entry..."), IconTheme.JabRefIcons.CREATE_ENTRY, KeyBinding.CREATE_ENTRY),
```

Comment:
"Create" and "Add" both used. Please use only "Create" (with the hope that native speakers are fine with this)

Reply:
We had an internal developer discussion:

- "Add entry"
- "Add entry using..."

And rename variables accordingly - should be easy with IntelliJ refactoring feature.

For the first one, we neglected "Add default entry" and "Add entry using defaults" (too long)

We think, we don't need an [architectural decision record](https://github.com/adr/madr/blob/develop/template/adr-template.md).

Sorry for the extra effort -- but  naming is hard in computer science: https://martinfowler.com/bliki/TwoHardThings.html